GitHubRepositories.ps1

```diff
@@ -15,6 +15,29 @@
      Set-Variable -Scope Script -Option ReadOnly -Name $_.Key -Value $_.Value
  }
 
+$squashMergeCommitTitleConversion = @{
```

Comment:
From a module style perspective, the preference is that all script-level variables are used the following way:

* References to them should explicitly include the `$script:` to make it clear where they come from
* Should be `SentenceCased`
* The should be read-only unless there's a good reason otherwise (e.g. being used to cache data)

We have the convenience creator at the top of the file to make those read-only script-level variables.  So, you can do something like this in there for these:


```powershell
{
    # ... cut for brevity
    GitHubRepositoryTeamPermissionTypeName = 'GitHub.RepositoryTeamPermission'
    SquashMergeCommitTitleConversion = @{
        'PRTitle' = 'PR_TITLE'
        'CommitOrPRTitle' = 'COMMIT_OR_PR_TITLE'
        'MergeMessage' = 'MERGE_MESSAGE'
    }
    # ... and the next one, etc...
 }.GetEnumerator() | ForEach-Object {
     Set-Variable -Scope Script -Option ReadOnly -Name $_.Key -Value $_.Value
 }
```

Reply:
Done.